test/functional/main_test.go

```diff
@@ -124,17 +151,23 @@ func TestMain(m *testing.M) {
 		logrus.Warningf("cleaned up left over uVMs: %s", strings.Split(s, "\r\n"))
 	}
 
+	// delete downloaded layers; cant use defer, since os.exit does not run them
+	for _, l := range images {
+		// just ignore errors: they are logged, and no other cleanup possible
+		_ = l.Close(context.Background())
+	}
```

Comment:
If the context ends, these would not be closed then right?

Reply:
`"wclayer".DestroyLayer` uses the context for logging, so theres no check that the context is cancelled.